### PR TITLE
Synstore tool now returns synapse id as output, added set-annotations and get-annotations tools

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: synapse-get-annotations-tool.cwl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+#!/usr/bin/env cwl-runner
+#
+# Authors: Andrew Lamb
```
Comment (kdaily) [inline synapse-get-annotations-tool.cwl]: please add `dct:creator` tags in the get and set annotations (or schema.org `Person` as in the [CWL user guide](https://www.commonwl.org/user_guide/17-metadata/index.html)).

File: synapse-get-annotations-tool.cwl [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  - id: synapse_config
+    type: File
+
+  - id: synapseid
+    type: string
+    inputBinding:
+      prefix: --id
+
+  - id: output_file_name
```
Comment (kdaily) [inline synapse-get-annotations-tool.cwl:33]: I know the command line tool provides this, but use `stdout` instead as it's a more standard interface.

File: synapse-store-tool.cwl [hunk 3]
```diff
@@ -49,3 +50,10 @@ outputs:
     type: File
     outputBinding:
       glob: stdout.txt
+
+  - id: file_id
```
Comment (kdaily) [inline synapse-store-tool.cwl:75]: Add a comment here to explain the regex and expected output of the `synapse store` command.

**Review by kdaily — CHANGES_REQUESTED** [review]
Some minor changes but looks good overall! Thank you for writing these!

File: synapse-get-annotations-tool.cwl (hunk 1 above)
Comment (kdaily) [reply synapse-get-annotations-tool.cwl]: I was having trouble using the old format I had seen in some CWL using `dct:creator`. The schema.org `Person` works for me.

**Review by kdaily — APPROVED** [review]
@andrewelamb add in your [ORCID identifier](https://orcid.org/) when you get one!

File: synapse-get-annotations-tool.cwl [hunk 4]
```diff
@@ -0,0 +1,59 @@
+
+#!/usr/bin/env cwl-runner
+
+$namespaces:
+  s: https://schema.org/
+  edam: http://edamontology.org/
```
Comment (kdaily) [inline synapse-get-annotations-tool.cwl]:
```suggestion
```

You don't use the edam namespace.

**Review by kdaily — CHANGES_REQUESTED** [review]

File: synapse-get-annotations-tool.cwl [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: 
+  - synapse
+  - get-annotations
+
+hints:
+  DockerRequirement:
+    dockerPull: sagebionetworks/synapsepythonclient:v1.9.2
```
Comment (thomasyu888) [inline synapse-get-annotations-tool.cwl]:
```suggestion
    dockerPull: sagebionetworks/synapsepythonclient:v2.1.0
```

File: synapse-set-annotations-tool.cwl [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: 
+  - synapse
+  - set-annotations
+
+hints:
+  DockerRequirement:
+    dockerPull: sagebionetworks/synapsepythonclient:v1.9.2
```
Comment (thomasyu888) [inline synapse-set-annotations-tool.cwl]:
```suggestion
    dockerPull: sagebionetworks/synapsepythonclient:v2.1.0
```

File: synapse-store-tool.cwl [hunk 7]
```diff
@@ -10,12 +10,19 @@ cwlVersion: v1.0
 $namespaces:
   dct: http://purl.org/dc/terms/
   foaf: http://xmlns.com/foaf/0.1/
+  s: https://schema.org/
 
 dct:creator:
   "@id": "https://orcid.org/0000-0001-5729-7376"
   foaf:name: Kenneth Daily
   foaf:mbox: "mailto:kenneth.daily@sagebionetworks.org"
```
Comment (thomasyu888) [inline synapse-store-tool.cwl]:
```suggestion
s:author:
  - class: s:Person
    s:identifier: https://orcid.org/0000-0001-5729-7376
    s:email: kenneth.daily@sagebionetworks.org
    s:name: Kenneth Daily
```

File: synapse-store-tool.cwl [hunk 8]
```diff
@@ -10,11 +10,19 @@ cwlVersion: v1.0
 $namespaces:
   dct: http://purl.org/dc/terms/
   foaf: http://xmlns.com/foaf/0.1/
```
Comment (thomasyu888) [inline synapse-store-tool.cwl:12]:
```suggestion
  dct: http://purl.org/dc/terms/
  foaf: http://xmlns.com/foaf/0.1/
```

**Pull request merged**